src/azure-cli/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+
+**Core**
```

Comment:
For the moment, we're still distributing `azure-cli-core` as a separate library. (Could change if we can get @lmazuel what he needs else-where, say the MSAL library) Until then, I'd add this message to `src/azure-cli-core/HISTORY.rst` instead of here.

Reply:
(Sorry, I know this is boring review feedback)

Reply:
I included a modified message in both.